package.json

```diff
@@ -6,18 +6,21 @@
     "lib",
     "README.md"
   ],
+  "main": "lib/index.js",
   "engines": {
     "node": ">=6.0.0"
   },
   "bin": {
-    "run-elm": "lib/index.js"
+    "run-elm": "lib/cli.js"
   },
   "scripts": {
-    "test:clean": "cd test/integration && git clean -d -f -X",
+    "clean:test": "git clean -d -f -X test/integration",
     "lint": "eslint src test --ignore-pattern elm-stuff",
+    "prebuild": "rimraf lib",
+    "build": "babel src --out-dir lib",
+    "postbuild": "cpx \"src/*.elm.template\" lib",
```

Comment:
&& are not allowed in npm scripts on Windows, so I had to split command into three